openapi/static/description/index.html

```diff
@@ -255,6 +255,36 @@ <h2 class="oe_slogan">Choose a platform that fit your needs</h2>
     </div>
 </section>
 
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+
+            <div class="alert alert-success oe_mt32" style="padding:0.3em 0.6em; font-size: 150%;">
+                <i class="fa fa-linode" aria-hidden="true"></i><b> RESTAPI/OPENAPI vs RPC </b>
+                <ul class="list-unstyled"><i>Integrated RPC functionality differs from Our implementation of RESTAPI/OPENAPI</i>:
```

Comment:
Our implementation of RESTAPI/OPENAPI has the following advantages over the built-in RPC functionality